ticdc/ticdc-changefeed-config.md

```diff
@@ -36,345 +36,573 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
+- This configuration item only takes effect if the downstream is a storage service.
+- Default value: `day`, which means separating files by day
+- Value options: `none`, `year`, `month`, `day`
+
+#### `enable-partition-separator`
+
+- Controls whether to use partitions as the separation string.
+- This configuration item only takes effect if the downstream is a storage service.
+- Default value: `true`, which means that partitions in a table are stored in separate directories
+- It is recommended that you keep the value as `true` to avoid potential data loss in downstream partitioned tables [#8581](https://github.com/pingcap/tiflow/issues/8581). For usage examples, see [Data change records](/ticdc/ticdc-sink-to-cloud-storage.md#data-change-records).
+
+#### `debezium-disable-schema`
+
+- Controls whether to disable the output of schema information.
+- Default value: `false`, which means enabling the output of schema information
+- This parameter only takes effect when the sink type is MQ and the output protocol is Debezium.
+
+#### sink.csv <span class="version-mark">New in v6.5.0</span>
+
+Starting from v6.5.0, TiCDC supports saving data changes to storage services in CSV format. Ignore the following configurations if you replicate data to MQ or MySQL sinks.
+
+##### `delimiter`
+
+- Specifies the character used to separate fields in the CSV file. The value must be an ASCII character.
+- Default value: `,`
+
+##### `quote`
+
+- Specifies the quotation character used to surround fields in the CSV file. If the value is empty, no quotation is used.
+- Default value: `"`
+
+##### `null`
+
+- Specifies the character displayed when a CSV column is NULL.
+- Default value: `\N`
+
+##### `include-commit-ts`
+
+- Controls whether to include commit-ts in CSV rows.
+- Default value: `false`
+
+##### `binary-encoding-method`
+
+- Specifies the encoding method of binary data.
+- Default value: `base64`
+- Value option: `base64`, `hex`
+
+##### `output-handle-key`
+
+- Controls whether to output handle key information. This configuration parameter is for internal implementation only, so it is not recommended to set it.
+- Default value: `false`
+
+##### `output-old-value`
+
+- Controls whether to output the value before the row data changes. The default value is false. 
+- When it is enabled (setting it to `true`), the UPDATE event will output two rows of data: the first row is a DELETE event that outputs the data before the change; the second row is an INSERT event that outputs the changed data.
+- When it is enabled, the `"is-update"` column will be added before the column with data changes. This added column is used to identify whether the data change of the current row comes from the UPDATE event or the original INSERT/DELETE event. If the data change of the current row comes from the UPDATE event, the value of the `"is-update"` column is `true`. Otherwise, it is `false`.
```

Comment:
```suggestion
- When it is enabled, the `"is-update"` column will be added before the column with data changes. This added column is used to identify whether the data change of the current row comes from the` UPDATE` event or the original `INSERT` or `DELETE` event. If the data change of the current row comes from the `UPDATE` event, the value of the `"is-update"` column is `true`. Otherwise, it is `false`.
```